kafka-webview-ui/src/main/resources/config/dev.yml

```diff
@@ -2,6 +2,9 @@ server:
   servlet:
```

Comment:
So the way I've structured the configuration files (which may or may not be a good way...) is `resources/config/base.yml` gets loaded with all of the "defaults" and then `dev.yml` gets layered ontop with any overrides.  

When building the ZIP file distributable,  this [config.yml](https://github.com/SourceLabOrg/kafka-webview/blob/master/kafka-webview-ui/src/assembly/distribution/config.yml) essentially takes the places of dev.yml.

That's the long winded way to say: If you think these things should be enabled by default, we likely want to move them down to `base.yml`.  Then we can add some documentation about how to configure/override the defaults for items we think could be relevant.  My goal hasn't been to completely re-document spring's internal configuration properties (an advance user can sort that out themselves), but instead just single out and document the more commonly configured properties as needed.

Clearly this is a work in progress, so feel free to keep things where you see fit, but just wanted to call it out :)

Reply:
Yeah we should definitely move some properties to config.yml.

Now that you mention configurations. I would suggest to follow Spring application property structure:
- resources/application.yaml
- resources/application-[PROFILE].yaml
This way we can load the right specific configuration through the property `spring.profiles.active`, that can be customised as a normal environment variable `SPRING_PROFILES_ACTIVE=blablabla` starting the container on Docker or Kubernetes.

I thought to propose you this refactoring after the Kubernetes series of PRs.

Reply:
Cool, yea lets do that after these set of changes 👍

Reply:
Moving some properties to `config.yml`, after that we can resolve this and continue with the PR review.

Reply:
Moved, closing this